mitodl/ocw-hugo-themes
# update default URLs to match new scheme

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/740

#### What's this PR do?
Recently, we updated the OCW sites to follow the following scheme:

RC:
Draft: https://draft-qa.ocw.mit.edu
Live: https://live-qa.ocw.mit.edu

Production:
Draft: https://draft.ocw.mit.edu
Live: https://ocw.mit.edu

This PR updates the default URLs used throughout the project for things like `STATIC_API_BASE_URL` and `RESOURCE_BASE_URL` to point at RC Live or Production Live where applicable.

#### How should this be manually tested?
 - Set up your `.env` based on the example `.env.example`
 - Make sure your local git client is authenticated with github.mit.edu
 - Spin up `ocw-www` with `npm run start:wwww`
 - Make sure `ocw-www` is accessible at https://localhost:3000
 - Spin up a test course with `npm run start:course`
 - Make sure your couse is up and running at https://localhost:3000
 - Check over the URLs used in the project and make sure that everything makes sense as to where it's pointing
